ocs_ci/ocs/ui/views.py

```diff
@@ -172,16 +197,22 @@
 }
 
 locators = {
+    "4.8": {
+        "login": login,
+        "page": page_nav,
+        "deployment": {**deployment, **deployment_4_7},
```

Comment:
I'd add pvc and infra dictionaries here as well, just to make sure that all the tests that can pass without any changes would pass.

Reply:
https://github.com/red-hat-storage/ocs-ci/blob/f2aadb01fcf596a0736caf8672d267da49399fd6/ocs_ci/ocs/ui/views.py#L174
there is not conflict on this pr.. I added a new key [4.8]

Reply:
Just for anyone who might also be interested, we discussed this issue with Oded separately and he explained that pvc tests are failing for 4.8 so they require a separate effort and thus the pvc dictionary was not included in this PR. As for infra dictionary, it needs to be tested as well.